.editorconfig

```diff
@@ -1,5 +1,12 @@
+root = true
```

Comment:
can we leave out all the linting and editor changes ? It makes the PR so hard to follow. I'm trying to debug the issues I found but its hard because you cant exactly pinpoint what changed due to the large amount of unrelated changes due to the linter preferences.

My proposal would be to create one PR for each functional change (or combine some small ones ofc) and one or more PR's for linter changes only otherwise. Also some feedback to why the linter changes are needed are also welcome as I see there are now some rules disabled which were previously enabled and we're further drifting apart from the baseline vue3/vuetify + vite config. My personal preference is to follow defaults as much as we can that comes with the language or tool.

Reply:
We had already talked about the linter several times and how useful it is, at that time I introduced it into the project.
The configuration is based on today's standards and recommendations. Not many rules were deactivated, the plug-ins add their own rules and not all of them are desired. There are more rules, which is not in proportion to the deactivations.

An undo at this point is not possible given the number of changes. Tell me what you want to quake and I'll do it. In addition, my code will not be overwritten and the work will not be disturbed. After all, you have given me the main responsibility for the frontend, so I can fix the errors or make the desired changes more quickly and cleanly.